test/support/event_store_case.ex

```diff
@@ -0,0 +1,25 @@
+defmodule Trento.EventStoreCase do
+  @moduledoc """
+  This module defines the test case to be used by
+  tests that require setting an event store.
+  """
+  use ExUnit.CaseTemplate
+
+  setup do
+    :ok = Application.stop(:trento)
+    :ok = Application.stop(:commanded)
+    :ok = Application.stop(:eventstore)
+
+    config = Trento.EventStore.config()
+    {:ok, conn} = Postgrex.start_link(config)
+    reset_event_store(conn, config)
```

Comment:
Disclaimer: I am ok with this.

just a nitpick: is this for readability?
I mean `reset_event_store` instead of just `EventStore.Storage.Initializer.reset!(conn, config)`

Reply:
ah yeah, refactoring stuff i missed that part, I was initializing the connection in that function and then I moved it on top to pass the connection in the context. i will fix it.